features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Throttling/Advanced/AddEdit.jsx

```diff
@@ -163,6 +163,7 @@ function AddEdit(props) {
                 }
                 break;
             case 'description':
+                break;
```

Comment:
The case check for 'description' field can be removed entirely.

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9541/commits/2809e66ce7558218fa0dd2ac5e74a4e82ff5fb83